### PR TITLE
Added Recipe

### PR DESCRIPTION
**Pull request by SleepyScarecrow**
somehow i forgot to add a recipe to the loverskiss drink

# Description

Added a recipe for the lover's kiss drink

# Changelog

:cl: Tilkku
- fix: Lover's Kiss Recipe

### CODE REVIEW

File: Resources/Prototypes/Floof/Recipes/Reactions/drinks.yml [hunk 1]
```diff
@@ -198,6 +198,16 @@
   products:
     LemonPopsicle: 10
 
+- type: reaction
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Recipes/Reactions/drinks.yml:201]:
```suggestion
- type: reaction # FloofStation
```

**Review by FoxxoTrystan — APPROVED** [review]
In all honest, i would love that you move all of those files in /Floof/ Directories instead.

**Review by FoxxoTrystan — CHANGES_REQUESTED** [review]
Oopsy mean to request change, see suggestion.

**Comment by SleepyScarecrow** [comment]
They are in floof though?

**Comment by SleepyScarecrow** [comment]
> They are in floof though?

Resources/Prototypes/Floof/Recipes/Reactions/drinks.yml

**Pull request merged**